src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -49,18 +52,19 @@ internal void CalculateEffectivePackageSaveMode()
                 packageSaveModeValue = SettingsUtility.GetConfigValue(Settings, "PackageSaveMode");
             }
 
-            EffectivePackageSaveMode = PackageSaveModes.None;
             if (!string.IsNullOrEmpty(packageSaveModeValue))
             {
+                // Restore \ Install always extract files
+                EffectivePackageSaveMode = Packaging.PackageSaveMode.Files;
                 foreach (var v in packageSaveModeValue.Split(';'))
                 {
-                    if (v.Equals(PackageSaveModes.Nupkg.ToString(), StringComparison.OrdinalIgnoreCase))
+                    if (v.Equals(Packaging.PackageSaveMode.Nupkg.ToString(), StringComparison.OrdinalIgnoreCase))
```

Comment:
I think `Enum.TryParse` supports numeric values (which does obviously does not) 
e.g.

`Enum.TryParse("2", out v)` -> `Packaging.PackageSaveMode.Nuspec`. I'd rather not add new behavior here.